lib/resources/kernel_module.rb

```diff
@@ -37,13 +66,56 @@ def loaded?
       !found.nil?
     end
 
+    # @note maintain logcal agreement with 'disabled?' method
+    alias enabled? loaded?
+
+    def blacklisted?
+      if inspec.os.redhat? || inspec.os.name == 'fedora'
+        modprobe_cmd = "/sbin/modprobe --showconfig | grep blacklist | grep #{@module}"
+      else
+        modprobe_cmd = "modprobe --showconfig | grep blacklist | grep #{@module}"
+      end
+      cmd = inspec.command(modprobe_cmd)
+      cmd.exit_status.zero? ? cmd.stdout.delete("\n") : nil
```

Comment:
Can you explain what you're trying to do with this line (and ones similar to it)?

Usually, Ruby methods that end in a `?` are expected to return a boolean value. But here, you'll either be returning the stdout without "\n" characters, or nil.